src/repositories/live.repository.ts

```diff
@@ -0,0 +1,330 @@
+    const { body } = await this.client.request.send<LiveLikeCountResponseRootObject>({
+      url: `/api/v1/live/${broadcastId}/get_like_count/` + (likeTs === '' ? '' : `?like_ts=${likeTs}`),
+      method: 'GET',
+    });
+    return body;
+  }
+
+  public async resumeBroadcastAfterContentMatch(broadcastId: string): Promise<any> {
+    // TODO: test
+    const { body } = await this.client.request.send({
+      url: `/api/v1/live/${broadcastId}/resume_broadcast_after_content_match/`,
+      method: 'POST',
+      form: this.client.request.sign({
+        _csrftoken: this.client.state.cookieCsrfToken,
+        _uid: this.client.state.cookieUserId,
+        _uuid: this.client.state.uuid,
+      }),
+    });
+    return body;
+  }
+
+  public async getJoinRequestCounts(
+    broadcastId: string,
+    lastTotalCount: number = 0,
+    lastSeenTs: number = 0,
+    lastFetchTs: number = 0)
+    : Promise<LiveJoinRequestCountsResponseRootObject> {
+    const baseUrl = `/api/v1/live/${broadcastId}/get_join_request_counts/`;
+    const { body } = await this.client.request.send<LiveJoinRequestCountsResponseRootObject>({
+      url: `${baseUrl}?last_total_count${lastTotalCount}&last_seen_ts=${lastSeenTs}&last_seen_ts${lastFetchTs}`,
+      method: 'GET',
+    });
+    return body;
+  }
+
+  public getObsSettings(info: LiveCreateBroadcastResponseRootObject): LiveObsSettings {
+    const parts = info.upload_url.split(new RegExp(info.broadcast_id));
+    return { stream_url: parts[0], stream_key: info.broadcast_id + parts[1]};
+}
+
+  public async startBroadcast(broadcastId: string, sendNotifications: boolean): Promise<LiveStartBroadcastResponseRootObject> {
+    const { body } = await this.client.request.send<LiveStartBroadcastResponseRootObject>({
+      url: `/api/v1/live/${broadcastId}/start/`,
+      method: 'POST',
+      form: this.client.request.sign({
+        _csrftoken: this.client.state.cookieCsrfToken,
+        _uuid: this.client.state.uuid,
+        should_send_notifications: sendNotifications,
+      }),
+    });
+    return body;
+  }
+
+  public async endBroadcast(
+    broadcastId: string,
+    endAfterCopyrightWarning: boolean = false,
+  ): Promise<LiveEndBroadcastResponseRootObject> {
+    const { body } = await this.client.request.send<LiveEndBroadcastResponseRootObject>({
+      url: `/api/v1/live/${broadcastId}/end_broadcast/`,
+      method: 'POST',
+      form: this.client.request.sign({
+        _csrftoken: this.client.state.cookieCsrfToken,
+        _uid: this.client.state.cookieUserId,
+        _uuid: this.client.state.uuid,
+        end_after_copyright_warning: endAfterCopyrightWarning,
+      }),
+    });
+    return body;
+  }
+
+  /* 301 - moved */
+
+  public async comment(broadcastId: string, message: string): Promise<any> {
+    // 301 - moved?
+    const { body } = await this.client.request.send({
+      url: `/api/v1/live/${broadcastId}/comment`,
```

Comment:
You get `301` here because you forgot to place `/` in the end of url 😄 
It is nice that you decided to keep this functions in PR, together we can fix everything